src/service/resources-impl.js

```diff
@@ -186,6 +189,15 @@ export class ResourcesImpl {
     /** @const @private {!Array<!Element>} */
     this.elementsThatScrolled_ = [];
 
+    /** @const @private {boolean} */
+    this.buildWhenCloseToViewport_ = isExperimentOn(
```

Comment:
Nit: `onlyBuildWhenNearViewport`

The "only" clarifies that this is restrictive rather than supplementary.